src/platform/user/profile/vet360/components/base/Vet360Transaction.jsx

```diff
@@ -22,15 +22,18 @@ function Vet360Transaction(props) {
   } = props;
 
   const method = transactionRequest?.method || 'PUT';
-  const hasError = isFailedTransaction(transaction);
+  const updateInProgress =
+    isPendingTransaction(transaction) || transactionRequest?.isPending;
+  const hasError =
+    isFailedTransaction(transaction) || transactionRequest?.isFailed;
```

Comment:
`isFailed` 😅

Reply:
oo I should double check this. we are using it elsewhere, but we also use `transactionRequest.error` 🤔

Reply:
Looking at [this test](src/platform/user/profile/vet360/tests/reducers/index.unit.spec.js), it seems we should have both `isFailed` and `error` available to us 😃 Not sure why there was ever a need to add `isFailed`.